plugins/components/conversionlayer_test.go

```diff
@@ -20,11 +19,7 @@ func TestCreateCommandUsages(t *testing.T) {
 	strFlag := NewStringFlag("flag", "", SetMandatory())
 
 	override := []string{"usage override", "usage override 2", "usage override 3"}
-	expectedOverride := []string{
-		fmt.Sprintf("%s %s", coreutils.GetCliExecutableName(), "usage override"),
-		fmt.Sprintf("%s %s", coreutils.GetCliExecutableName(), "usage override 2"),
-		fmt.Sprintf("%s %s", coreutils.GetCliExecutableName(), "usage override 3"),
-	}
+	expectedOverride := override
```

Comment:
If `expectedOverride` is assigned the value of `override`, why are the two variables needed?